sycl/include/sycl/detail/type_traits/vec_marray_traits.hpp

```diff
@@ -13,6 +13,18 @@
 
 #include <sycl/detail/defines_elementary.hpp>
 
+#ifndef __SYCL_USE_PREVIEW_VEC_IMPL
+#if defined(__INTEL_PREVIEW_BREAKING_CHANGES)
+// Several specification changes need to be implemented together to keep CTS
+// passing. We'll switch to `1` once they all land.
+// `__SYCL_USE_PLAIN_ARRAY_AS_VEC_STORAGE` needs to be changed to use this
+// `__SYCL_USE_PREVIEW_VEC_IMPL` at that time as well.
+#define __SYCL_USE_PREVIEW_VEC_IMPL 0
+#else
+#define __SYCL_USE_PREVIEW_VEC_IMPL 0
```

Comment:
@AlexeySachkov , any better suggestions for the naming here?

Reply:
One issue is that we want to preserve current/"old" version *after* next major release, when the meaning of "preview" would change. On the other hand, we can rename the macro at that time, maybe.

Reply:
> One issue is that we want to preserve current/"old" version _after_ next major release, when the meaning of "preview" would change. On the other hand, we can rename the macro at that time, maybe.

If we plan to expose the macro to users so that they could do a switch back (for whichever reason), then it is better not to rename it later, but agree on a name from the beginning - otherwise renaming would be an API change by itself.

What about something like `__SYCL_USE_SYCL2020_CONFORMANT_IMPL`? I.e. indicate the intent instead of old/new where the meaning of old/new changes in time.

Reply:
@AlexeySachkov , can you take another look?